### ISSUE TITLE
instead of specifying objectName's in SecretproviderClass it would be great to read all secrets in key-vault and mount in the pod

### ISSUE THREAD

**Issue by skri547** [Azure/secrets-store-csi-driver-provider-azure#477]
**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Secrets Store CSI Driver version: (use the image tag):
- Azure Key Vault provider version: (use the image tag):
- Kubernetes version: (use `kubectl version`):
- Cluster type: (e.g. AKS, aks-engine, etc):

**Comment by skri547** [comment]
Is there any solution already in place? If yes can you provide the relevant pods or steps to achieve this? 
Thanks

**Comment by skri547** [comment]
@aramase any solution present already for this?

**Comment by aramase** [comment]
@skri547 No, that's not possible. The objects and versions (latest if not set) defined in the `SecretProviderClass` are the only ones fetched from keyvault.
